src/api/comment/content-types/comment/lifecycles.js

```diff
@@ -19,23 +19,32 @@ module.exports = {
     const idea = comment.idea_card;
     const user = comment.user;
 
-    await strapi.entityService.create("api::subscription.subscription", {
-      data: {
-        entityType: "Comment",
-        entityId: commentId,
-        createdDateTime: new Date(),
-        active: true,
-        user: user,
-      },
-    });
+    /** verify that user isn't already subscribed to ideaCard */
+    const existingSubscriptions = await strapi.entityService.findMany(
```

Comment:
Do we need to find all of them? Or is `findOne` enough?

Reply:
yes one is enough, i'll limit it to one entry